x-pack/plugins/uptime/public/components/monitor/ml/manage_ml_job.tsx

```diff
@@ -61,8 +61,13 @@ export const ManageMLJobComponent = ({ hasMLJob, onEnableJob, onJobDelete }: Pro
       disabled={hasMLJob && !canDeleteMLJob}
       isLoading={isMLJobCreating || isMLJobLoading}
       size="s"
+      aria-label={labels.ENABLE_MANAGE_JOB}
     >
-      {hasMLJob ? labels.ANOMALY_DETECTION : labels.ENABLE_ANOMALY_DETECTION}
+      {isMLJobCreating || isMLJobLoading
```

Comment:
The readability of this line could be a bit better. Maybe extract it to a module-level function outside of the component we can call and separate some of this logic?

Reply:
i have tried to improved it